huaweicloud/services/acceptance/obs/resource_huaweicloud_obs_bucket_test.go

```diff
@@ -593,6 +648,20 @@ resource "huaweicloud_obs_bucket" "bucket" {
 `, randInt)
 }
 
+func testAccObsBucketConfigWithStorageInfo(randInt int) string {
+	return fmt.Sprintf(`
+resource "huaweicloud_obs_bucket" "bucket" {
+  bucket = "tf-test-bucket-%d"
+  acl    = "private"
+
+  storage_info {
+    object_number = 0
+    size          = 0
+  }
+}
+`, randInt)
+}
```

Comment:
There are also default values for integer. I think you can pick this part into basic test for null value.

Reply:
done